### PR TITLE
Two item meta info

### PR DESCRIPTION
**Pull request by SebastianGerS**
I made this pull request mostly to get some feedback.
I added the image to, just to see how it would look so basically this is the entire page for a single item
Ofcourse some logic for when seasons are displayed or not will be added when we start to use the db info
not that I have not done any adaptions for smaller screens yet.

### CODE REVIEW

File: resources/assets/sass/components/item_meta_info.scss [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+  #hero-header {
+    font-family: $heading-font;
+    font-size: $title-hero-font-size
+  }
+
+}
+
+.item-meta-info {
+
```
Comment (AnnaRoss) [inline resources/assets/sass/components/item_meta_info.scss]: An empty line has slipped in here :smile:

File: resources/assets/sass/components/item_meta_info.scss [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  font-size: $paragraph-font-size;
+
+  .title-genres {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    font-size: $sub-heading-font-size;
+    color: $heading-alt-color;
+
```
Comment (AnnaRoss) [inline resources/assets/sass/components/item_meta_info.scss]: An empty line has slipped in here 😄

File: resources/views/layouts/item_meta_info.blade.php [hunk 3]
```diff
@@ -1,4 +1,27 @@
...
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.1/css/bulma.min.css">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <!-- CSRF Token -->
+      <meta name="csrf-token" content="{{ csrf_token() }}">
+      <title>{{ config('app.name', 'Laravel') }}</title>
+      <!-- Styles -->
+      <link href="{{asset('css/item_meta_info.css')}}" rel="stylesheet">
+   </head>
+  </body>
```
Comment (AnnaRoss) [inline resources/views/layouts/item_meta_info.blade.php]: This should be an opening body tag instead?

File: resources/views/layouts/components/item_meta_info.blade.php [hunk 4]
```diff
@@ -12,83 +12,85 @@
...
+                  <tr>
+                    <td span="2"><a>2</a></td>
+                    <td span="2">21</td>
+                    <td span="2">1998</td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+            <div class="row-flex-start">
+              <h3>Short</h3><h2>Facts</h2>
```
Comment (AnnaRoss) [inline resources/views/layouts/components/item_meta_info.blade.php]: I think it's a bit of a misuse to use `<h3>` and `<h2>` elements in this way. Maybe we could just make it :
```html
<h2>short<strong>Facts</strong></h2>
```
That isn't very semantically correct either. Just a thought. 

Or perhaps that we make use of the `<caption>` tag:
```html
<table>
      <caption>Short Facts</caption>
</table>
```
Or just some more lovely `<span>` elements. 
We could also just let go of how it looks in the prototype.

**Review by AnnaRoss — APPROVED** [review]
Some general comments: 

* Maybe you could insert empty lines between significant elements in the item_meta_info.blad.php. For example:
```
<body>

     <div></div>
```
```
</table>

<div class="plot-summary">
```
If it is possbile to exchange some of the `<div>` tags into `<section>` it would be great - thinking about a11y and _search engine optimisation_. But I don't know if it would affect the current css too much - it is not a priority! Most importantly - it works.

File: resources/assets/sass/components/item_meta_info.scss (hunk 1 above)
Comment (SebastianGerS) [reply resources/assets/sass/components/item_meta_info.scss]: should be fixed now, also restructured it a bit now always leaving one blank row before writing a property:value pair for a given selector

Comment (SebastianGerS) [reply resources/assets/sass/components/item_meta_info.scss]: but with html I realy like it when there's no blank rows at all for some wierd reasson, but I get your point it's probably more readable with some blank rows

File: resources/views/layouts/components/item_meta_info.blade.php (hunk 4 above)
Comment (SebastianGerS) [reply resources/views/layouts/components/item_meta_info.blade.php]: I changed it to 
`<h2><span>Short</span><span>Facts</span></h2>`

and regarding the other semantic stuff I changed some of the div to other types of elements, seams to work fine : ) 
Hard to determine how to correctly use sections and articles thought, now I have some sections in sections and stuff like that....

**Review by ghost — APPROVED** [review]
Looks good

**Pull request merged**